### ISSUE TITLE
It cost more than 5 minutes the network detect a brand new user for V0.3.0

### ISSUE THREAD

**Issue by ticktackim** [ticktackim/ticktack-workplan#92]
Now the new client can link with the autopub no problem.But it costs more than 5 minutes the network detect a brand new user for V0.3.0.

**Comment by dominictarr** [comment]
yes I am currently working on improved performance for initial sync, my goal for it to take far less than 5 minutes.
